next/pages/search.tsx

```diff
@@ -116,12 +124,17 @@ export default function SearchPage() {
   );
 }
 
-export const getStaticProps: GetStaticProps<CommonPageProps> = async (
+export const getStaticProps: GetStaticProps<SearchPageProps> = async (
   context
 ) => {
+  // We have to pass the path of the page manually here, it would be better to infer it
+  // automatically, but apparently this is not doable in this context.
+  const languageLinks = createLanguageLinksForNextOnlyPage("/search", context);
```

Comment:
it's uncool that we have to hadcode the path here, since it comes from the filename anyway, but apparently there's no way to get to that information in this context.